web/src/lib/hooks.ts

```diff
@@ -278,6 +278,7 @@ const MODEL_DISPLAY_NAMES: { [key: string]: string } = {
   "claude-2.0": "Claude 2.0",
   "claude-instant-1.2": "Claude Instant 1.2",
   "claude-3-5-sonnet-20240620": "Claude 3.5 Sonnet",
+  "claude-3-5-sonnet-20241022": "Claude 3.5 Sonnet (New)",
```

Comment:
seems possible we might need to rename (new) later but i suppose that's easily fixable if it happens